### DISCUSSION TITLE
How to update to latest preview packages for Blazor Web App

### DISCUSSION THREAD

**Discussion by mohaaron** [dotnet/aspnetcore#49452] [Q&A]
Hello all,

I've just updated to VS 2022 17.7 Preview 3, and have dotnet version 8.0.100-preview.6.23330.14 installed. I'm now wondering how can I know what preview version of the packages are referenced in a Blazor Web App (SSR) that I have created. I see that there are no specific packages shown as dependencies in the project and when I open the nuget manager I see a list of packages I can update to, but it also doesn't show any packages as being installed. How would I go about updating to the latest packages when there are non to update?

**Comment by casbeebc** [comment]
Have you tried enabling preview features in Tools menu:
https://anthonygiretti.com/2023/06/10/net-8-why-net-8-preview-doesnt-show-up-in-visual-studio-2022/

**Comment by mohaaron** [comment]
I'm now updated to Visual Studio 17.8 Preview 1. In the Preview version of Visual Studio there is no option for Use previews of the .NET SDK. Probably because it's a preview of Visual Studio that option is unnecessary. Thanks for the comment though. I did go and check that in the release version of VS that I have.
